controllers/selfnoderemediation_controller.go

```diff
@@ -81,6 +83,9 @@ var (
 		Value:  "nodeshutdown",
 		Effect: v1.TaintEffectNoExecute,
 	}
+
+	// OutOfServiceTimeoutDuration - time after which out-of-service taint is automatically removed
+	OutOfServiceTimeoutDuration = 1 * time.Minute
```

Comment:
Shouldn't it be a configurable value?

Reply:
No strong opinion, we can expose the configuration later, eventually 👍